evap/evaluation/templates/django/forms/widgets/checkbox.html

```diff
@@ -1 +1,6 @@
-<input type="{{ widget.type }}" name="{{ widget.name }}"{% if widget.value != None %} value="{{ widget.value|stringformat:'s' }}"{% endif %}{% include 'django/forms/widgets/attrs.html' %} />
+<div class="form-check">
+    <input class="form-check-input" type="{{ widget.type }}" name="{{ widget.name }}"
+        {% if widget.value != None %} value="{{ widget.value|stringformat:'s' }}"{% endif %}
+        {% include 'django/forms/widgets/attrs.html' %} />
+    <label class="form-check-label"></label>
```

Comment:
"for" missing on purpose?

Reply:
this is always empty, so the for is not needed. i'll add it anyways to be consistent.